src/GraphQL.ApiTests/BomTests.cs

```diff
@@ -10,10 +10,12 @@ public void Files_Should_Not_Use_BOM()
     {
         string GetPath([CallerFilePath] string path = "") => path; // <GIT_ROOT>\src\GraphQL.ApiTests\BomTests.cs
 
-        var gitRoot = new DirectoryInfo(GetPath()).Parent!.Parent!.Parent!;
+        var callerFilePath = GetPath();
+        var gitRoot = new FileInfo(callerFilePath).Directory!.Parent!.Parent!;
```

Comment:
This change is a bug fix in our tests.  `GetPath()` returns a file not a directory.

Reply:
(the comment in line 11 is still an accurate representation of what is returned from `GetPath()`)

Reply:
Thanks!